Test/QuickCheck/Modifiers.hs

```diff
@@ -343,4 +355,100 @@ instance (Arbitrary a, ShrinkState s a) => Arbitrary (Shrinking s a) where
 #endif /* NO_MULTI_PARAM_TYPE_CLASSES */
 
 --------------------------------------------------------------------------
+-- | @ASCII@.
+newtype ASCII = ASCII {getASCII :: Char}
+  deriving ( Eq, Ord, Show, Read )
+
+asciiGen :: Gen Char
+asciiGen = choose ('\0', '\127')
+
+instance Arbitrary ASCII where
+  arbitrary = ASCII `fmap` asciiGen
+
+--------------------------------------------------------------------------
+-- | @Latin1@: latin1 Char, same Arbitrary as Char itself.
+newtype Latin1 = Latin1 {getLatin1 :: Char}
```

Comment:
I think it's just a historical accident that `Arbitrary Char` only generates Latin1. We should just bite the bullet and change it to Unicode.

Is a Latin1 generator useful in its own right? Or would it be enough with Unicode and (7-bit) ASCII?